doc/index.rst

```diff
@@ -204,17 +204,19 @@ Data download and generation
    datasets.fetch_traffic_violations
    datasets.fetch_drug_directory
    datasets.fetch_world_bank_indicator
+   datasets.get_ken_table_aliases
+   datasets.get_ken_types
    datasets.get_ken_embeddings
    datasets.get_data_dir
    datasets.make_deduplication_data
 
 About
 =====
 
-dirty_cat is for now a repository for ideas coming out of a research
+dirty_cat is currently a repository for ideas coming out of a research
 project: there is still little known about the problems of dirty
 categories. Tradeoffs will emerge in the long run. We really need people
-giving feedback on success and failures with the different techniques and
+giving feedback on successes and failures with the different techniques and
```

Comment:
How about we rewrite this paragraph: "dirty_cat is a young project. We really need people ..."